src/tsickle.ts

```diff
@@ -745,11 +761,18 @@ class Annotator extends ClosureRewriter {
           typeCopy.types = nonNullUnion;
           type = typeCopy;
         }
+        // See comment above.
+        if (this.templateSpanStackCount > 0) {
+          this.emit('(');
+        }
         this.emitJSDocType(nnexpr, undefined, type);
         // See comment above.
         this.emit('((');
```

Comment:
We still need the double paren thing for same cases sadly (I saw it via the golden tests) :-(